committeeoversightapp/templates/committeeoversightapp/category_detail_page.html

```diff
@@ -21,7 +21,7 @@ <h2 class="mb-4 text-center">Category: {{page.category}}</h2>
 
   <div class="card mt-5 col-12 col-xl-10 mx-auto card-shadow">
     <div class="card-body">
-      {% include 'partials/hearing_table.html' %}
+      {% include 'partials/hearing_table.html' with table_size="table-sm" %}
```

Comment:
The hearings table on the Browse Hearings page doesn't need to be as compact as it is on the Committee and Category detail pages.